tests/infra/network.py

```diff
@@ -802,12 +802,14 @@ def wait_for_node_commit_sync(self, timeout=3):
             for node in self.get_joined_nodes():
                 with node.client() as c:
                     r = c.get("/node/commit")
-                    commits.append(f"{r.view}.{r.seqno}")
+                    assert r.status_code == http.HTTPStatus.OK.value
```

Comment:
This function `wait_for_node_commit_sync` has been broken for a while, because `/node/commit` no longer returns view+seqno in headers, so we were constructing a list of `None.None` strings which is identical enough to pass the check. This change extracts view and seqno from the JSON body.

Reply:
Thank you for catching that!